doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -199,21 +219,28 @@ Common data preparation steps: resampling, smoothing, filtering
 Resampling
 ..........
 
-:class:`NiftiMasker` and many similar classes enable resampling images.
+:class:`NiftiMasker` and many similar classes enable recasting of images
+       into different resolutions and constellations of brain voxel data.
```

Comment:
I find resampling clearer, but that may be me and the fact that I don't really understand what "constellations of brain voxel data" means.